src/components/header/buttons/PeersButton.js

```diff
@@ -41,7 +43,11 @@ export default class PeersButton extends Component {
   }
 
   onPeersChange () {
-    this.setState({ peers: this.props.peerGroup.all() })
+    if (this.props.peerGroup) {
+      this.setState({ peers: this.props.peerGroup.all() })
+    } else {
+      setTimeout(this.onPeersChange.bind(this), 1000)
```

Comment:
why do we need to rebind `onPeersChange` here? Would it be ok just to ignore the event if peerGroup is null?